test/test_api_io.ml

```diff
@@ -0,0 +1,34 @@
+let test ~name ~body ~expected_errors =
+  ( name
+  , `Quick
+  , fun () ->
+      let json = Yojson.Safe.from_string body in
+      let result = Cs_api_io.get_graphql_errors json in
+      Alcotest.(check (list string)) name result expected_errors )
+
+let body_ok =
+  "{\"Not an error\":[{\"message\":\"OK \
+   found\",\"locations\":[{\"line\":3,\"column\":9}],\"path\":[\"createTrace\"]}],\"data\":{\"createTrace\":null}}"
```

Comment:
Is the `"Not an error"` field realistic? I wonder if it's necessary here. When a query works, you usually have just the `data` field, right?

Reply:
It's not realistic, but I wanted to put something having a `message` field. I'll just leave the `data` field.